src/App.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import './App.css';
+import { TodoList } from './components/TodoList/TodoList';
 
 import todos from './api/todos';
 import users from './api/users';
 
+const preparedTodos = todos.map(todo => (
+  {
+    user: users.find(user => user.id === todo.userId),
+    ...todo,
+  }));
+
```

Comment:
очень странный перенос
```suggestion
const preparedTodos = todos.map(todo => ({
  user: users.find(user => user.id === todo.userId),
  ...todo,
}));

```